test/mp4-emsg.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Not a huge deal either, but it looks like most of the other files are indented with 2 spaces and this one is 4.